src/common/libidset/test/idset.c

```diff
@@ -154,6 +413,12 @@ int main (int argc, char *argv[])
     test_badparam ();
     test_codec ();
     test_codec_large ();
+    test_iter ();
+    test_set ();
+    test_range_set ();
+    test_clear ();
+    test_range_clear ();
+    test_copy ();
 
```

Comment:
i don't think i see a unit test for AUTOGROW?

Reply:
Thanks will add some.